### DISCUSSION TITLE
Will I ever be able to use Capacitor to design a web app for Apple Vision Pro?

### DISCUSSION THREAD

**Discussion by vesper8** [ionic-team/capacitor#7274] [General]
Title says it all. I'm seeing a lot more apps released now that support the AVP. Can I do that with a Capacitor app?

**Comment by taxilian** [comment]
It does seem like it should be pretty easy to add support for at least a basic level -- if I could just put my app in and have it so I could resize the window to any arbitrary size that would be a great start for me. However, when I try to build it for the vision pro it tells me that Capacitor isn't found, presumably because there is no build for vision pro published.

**Reply by fnc12** [reply]
same for me. When I try to specify `Capacitor` lib to be targeted for `:visionos` I get `pod install` error saying
```
[!] The platform of the target `my-target` (visionOS 1.2) is not compatible with `Capacitor (6.1.0)`, which does not support `visionOS`.
```

**Reply by vesper8** [reply]
This might help. Unsure why it isn't being merged https://github.com/ionic-team/capacitor/pull/7394

**Reply by vesper8** [reply]
Also this https://ionic.io/blog/augmented-reality-with-capacitor-ar-quick-look
